templates/rsync.conf.lua.erb

```diff
@@ -2,6 +2,13 @@ sync {
   default.rsync,
   source = "<%= @source %>",
   target = "<%= @target %>",
+  <%- if @delete != true -%>
+    <%- if @delete == false -%>
+  delete = <%= @delete %>,
+    <%- else -%>
+  delete = "<%= @delete %>",
+    <%- end -%>
+  <%- end -%>
```

Comment:
Hello Nic - I appreciate your willingness to look into it.

The reason for the first conditional was to cause no change in config files for existing configurations. As `true` is the default I opted to not print it then.

As for the second conditional, reading the [doc](https://axkibe.github.io/lsyncd/manual/config/layer4/#deletions) it seems to me the first two options are boolean, the former ones string.
   I just tested to include the modes `running/startup` unquoted and they do not follow their advertised behaviour - i.e. `startup` is not supposed to delete on target during runtime, but will do so if the settings is placed unquoted (as it resolves to `bool true`?). Same goes for printing `true` quoted, it will not delete on target. Maybe I should look into lsyncd if they can all be made strings.

If the control structure / indentation looks awkward or introduces confusion and you prefer any other way, please change. To me the distinction boolean/string-option proved to be important and therefore at least one conditional is needed.